From 31216cdcd064835aeaaf7bd20a1e181a8c0d53e7 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:15 -0400
Subject: [redhat] kabi: Adding symbol set_device_ro (block/genhd.c)

Message-id: <10f985a1b017d46f2d3b6ee35b5d59036ad93f1e.1410510773.git.poros@redhat.com>
Patchwork-id: 95997
O-Subject: [PATCH RHEL7.1 195/233] kabi: Adding symbol set_device_ro (block/genhd.c)
Bugzilla: 1062361
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Used by PowerPath for BLKROSET ioctl to set device read-only

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/set_device_ro
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/set_device_ro
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/set_device_ro

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/set_device_ro b/redhat/kabi/kabi-module/kabi_ppc64/set_device_ro
new file mode 100644
index 0000000..4b3dc84
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/set_device_ro
@@ -0,0 +1,2 @@
+#1-
+0x3387a2ea set_device_ro vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/set_device_ro b/redhat/kabi/kabi-module/kabi_s390x/set_device_ro
new file mode 100644
index 0000000..fb680b7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/set_device_ro
@@ -0,0 +1,2 @@
+#1-
+0xad5b9670 set_device_ro vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/set_device_ro b/redhat/kabi/kabi-module/kabi_x86_64/set_device_ro
new file mode 100644
index 0000000..d179312
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/set_device_ro
@@ -0,0 +1,2 @@
+#1-
+0x3ac5607e set_device_ro vmlinux EXPORT_SYMBOL
-- 
1.7.1